app/assets/stylesheets/pin-show.css.scss

```diff
@@ -54,6 +56,15 @@ button.back{
   }
 }
 
+#numLikes{
```

Comment:
* do you absolutely need an id as your css selector? 
* an id, like a class, should use spinal-case, not camelCase